tower/src/make/make_service.rs

```diff
@@ -247,3 +247,80 @@ where
         self.make.make_service(target)
     }
 }
+
+/// A [`MakeService`] that produces services by cloning an inner service.
```

Comment:
It would be nice to have examples here, IMO...maybe with `buffer` to make a service `Clone`?

Reply:
Sure thing. I haven't used `MakeService` much myself so not quite sure what a decent example might be? Any suggestions?

Reply:
I added an example in 57776bf8f6aad